src/Recepies/SearchResults/SearchResult.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SearchResults.scss';
+import { connect } from 'react-redux';
+
+
+export const SearchResult = ({ recipes, loading }) => {
+  return(
+    <div className="resultsWrapper">
+      <h4>Recipe Results</h4>
+      {
+        (recipes && (recipes.length >= 1)) ?
```

Comment:
Indeed, I have refactored and changed recipes to now a null value. This would improve code down the components where I would not need this statement `recipes && recipes.length`